plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -18,7 +18,13 @@ def user
     # Creates a new Kubernetes Release and corresponding ReleaseDocs
     def self.create_release(params)
       Kubernetes::Release.transaction do
-        release = create(params.except(:deploy_groups))
+        release = create(params.except(:deploy_groups)) do |release|
+          if params.fetch(:deploy_groups).any? { |dg| dg.fetch(:roles).any? { |role| role.fetch(:role).blue_green? } }
+            release.blue_green_color = begin
+              release.previous_successful_release&.blue_green_color == "blue" ? "green" : "blue"
```

Comment:
very minor: as a user, I'd expect the first deploy to be `blue` (to be consistent with the strategy name). What if we invert it comparing previous color to green?